root/relationship/linkattributetype/RelationshipAttributeTypesIndex.js

```diff
@@ -0,0 +1,181 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {upperFirst} from 'lodash';
+
+import {withCatalystContext} from '../../context';
+import Layout from '../../layout';
+import {compare} from '../../static/scripts/common/i18n';
+import expand2react from '../../static/scripts/common/i18n/expand2react';
+import bracketed, {bracketedText}
+  from '../../static/scripts/common/utility/bracketed';
+import RelationshipsHeader from '../RelationshipsHeader';
+
+type AttributeTreeProps = {
+  +$c: CatalystContextT,
+  +attribute: LinkAttrTypeT,
+};
+
+type AttributeDetailsProps = {
+  +$c: CatalystContextT,
+  +attribute: LinkAttrTypeT,
+  +topLevel?: boolean,
+};
+
+type AttributesListProps = {
+  +$c: CatalystContextT,
+  +root: LinkAttrTypeT,
+};
+
+function compareChildren(a: LinkAttrTypeT, b: LinkAttrTypeT): number {
+  if (a.child_order === b.child_order) {
+    return compare(a.name, b.name);
+  }
+  return a.child_order < b.child_order
+    ? -1
+    : (a.child_order > b.child_order ? 1 : 0);
+}
```

Comment:
`sort((a, b) => compareChildren(a, b))` is not equivalent to the TT `sorted_children(c.get_collator)` that used to follow user selected locale. That is the same sorting issue found in https://github.com/metabrainz/musicbrainz-server/pull/1431 but this time it is a regression.

Reply:
Hmm. https://github.com/metabrainz/musicbrainz-server/blob/f5f33f1271dff8f4f685da8f76b67882a19cd149/root/static/scripts/common/i18n.js seems to use documentLang - isn't that the same?

Reply:
Maybe it is just me but `/relationship-attributes` isn’t ordered the same than on beta.

Reply:
I wasn't actually using the translations for sorting at all because I'm dumb. Fixing!